guides/source/i18n.md

```diff
@@ -747,15 +747,15 @@ pluralization backend. By default, only the English pluralization rules are appl
 
 ```ruby
 I18n.backend.store_translations :en, inbox: {
-  zero: 'no messages', # optional
-  one: 'one message',
+  '0': 'no messages', # optional
```

Comment:
Tangential to your PR, but in my experience it's rare to call `store_translations` vs just putting translations in a YML file. So maybe this snippet should do that instead, so it more closely matches how users actually interact with i18n.